addons/mail/static/src/core/common/notification_message.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates xml:space="preserve">
+    <t t-name="mail.NotificationMessage">
+        <div class="o-mail-NotificationMessage text-break mx-auto text-muted opacity-75 px-3 text-center smaller" t-on-click="onClickNotificationMessage" t-ref="root">
+            <i t-if="message.notificationIcon" t-attf-class="{{ message.notificationIcon }} me-1"/>
+            <t t-if="message.notificationType === 'call'">
+                <span t-esc="callInformation"/> <span class="o-mail-Message-date o-xsmaller" t-esc="message.dateSimpleWithDay"/>
+            </t>
+            <t t-else="">
+                <span class="o-mail-NotificationMessage-author d-inline" t-if="message.authorName and !message.body.includes(escape(message.authorName))" t-esc="message.authorName"/> <t t-out="message.body"/>
```

Comment:
The date part has lower opacity than a message, due to `opacity-75` affecting also the date.

The `.opacity-75` should apply on content of notification message only. The date should have the exact same opacity as the message date.

Similar remark made in a relatively equivalent PR: https://github.com/odoo/odoo/pull/200343/files#r1994018265

This PR is very close to merge, so I would just fix this part, so the other PR can be added on top of this one.